core/src/main/java/org/infinispan/xsite/statetransfer/XSiteStateProviderImpl.java

```diff
@@ -163,10 +161,10 @@ private void sendFromSharedBuffer(XSiteBackup xSiteBackup, List<XSiteState> shar
       }
       XSiteState[] privateBuffer = sharedBuffer.toArray(new XSiteState[0]);
 
-      if (trace) {
+      if (log.isTraceEnabled()) {
          log.debugf("Sending chunk to site '%s'. Chunk contains %s", xSiteBackup.getSiteName(),
                     Arrays.toString(privateBuffer));
-      } else if (debug) {
+      } else if (log.isDebugEnabled()) {
```

Comment:
This is wrong IMO. We shouldn't log individual chunks at DEBUG level because the number of chunks depends on the number of entries (and the chunk size).